src/TorchSharp/TorchVision/dsets/BigEndianReader.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) .NET Foundation and Contributors.  All Rights Reserved.  See LICENSE in the project root for license information.
+using System;
+using System.IO;
+
+namespace TorchSharp.torchvision.dsets
+{
+    public class BigEndianReader
```

Comment:
Please add comments.

Reply:
@dayo05 -- ping! :-)

Reply:
I was focusing to other project, I;ve fixed that now